assignment3/src/problem1_UsingJS5.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Closure Demo</title>
+        <script>
+            function printNumbers() {
+                const SLEEP_TIME = 1000;
+                let counter = 1;
+                setInterval(function () {
```

Comment:
setInterval should be cleared. The program doesn't stop after printing 10 numbers.